Mage/src/main/java/mage/game/jumpstart/JumpstartPoolGenerator.java

```diff
@@ -42,46 +40,64 @@
    * themed land art created for the set. The booster pack is wrapped like a regular pack, but the set of cards is
    * packed in an additional plastic wrap, with a face card (the "Pack Summary card") that indicates the theme and
    * the color of the half-deck."
-   *
-   * Source:  https://mtg.gamepedia.com/Jumpstart
-   *
+     *
+     * Source: https://mtg.gamepedia.com/Jumpstart
+     *
    * Announcement: https://magic.wizards.com/en/articles/archive/news/introducing-jumpstart-new-way-play-magic-2020-02-20
    * Card Pool: https://magic.wizards.com/en/articles/archive/card-image-gallery/jumpstart
    * Deck Lists: https://magic.wizards.com/en/articles/archive/feature/jumpstart-decklists-2020-06-18
-   */
-
-  private static final String RESOURCE_NAME = "jumpstart/jumpstart.txt";
-  private static final List<JumpstartPack> JUMPSTART_PACKS;
+     */
+    private static final String RESOURCE_NAME = "jumpstart/jumpstart.txt";
+    private static List<JumpstartPack> JUMPSTART_PACKS;
```

Comment:
It's a static field, you can't store user defined data here (one upload will rewrite data for all on the server). You must use packs list per tourney/options.

Reply:
Modified back to what it was.  Hopefully resolved ok.